apps/studio/components/interfaces/SignIn/SignUpForm.tsx

```diff
@@ -47,6 +49,7 @@ const formId = 'sign-up-form'
 export const SignUpForm = () => {
   const captchaRef = useRef<HCaptcha>(null)
   const [showConditions, setShowConditions] = useState(false)
+  const [isSubmitting, setIsSubmitting] = useState(false)
```

Comment:
we should be able to get this value from `form.formstate.isSubmitting` instead of creating a new `useSttate` just FYI

Reply:
oh yes! great catch hahaha will update